spec/features/hub/toggle_client_bank_account_info_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
```

Comment:
Wondering if it would be enough to test this as part of an existing flow in the `show_client_spec` (i.e. click "View" as we're validating other client fields").

Reply:
The reason for breaking it out is that i need to require js for the test in order for it to work, and i wasnt sure if its just the SETUP for enabling js that takes a while or if it actually slows down the rest of the scenario. Since I couldn't tell, I tried to isolate it... not sure if that makes sense or makes it worse.